cpp/src/gandiva/tests/projector_test.cc

```diff
@@ -1752,6 +1752,112 @@ TEST_F(TestProjector, TestSign) {
   EXPECT_ARROW_ARRAY_EQUALS(out_float64, outputs4.at(0));
 }
 
+TEST_F(TestProjector, TestAbs) {
```

Comment:
Could you cleanup this like the followings?

* https://github.com/apache/arrow/pull/13656/commits/7d866b5a6e3589e619b042fcde37a18e87fd9121
* https://github.com/apache/arrow/pull/13656/commits/7918e72b5f921ad5f0ac22a5f048c423ccac4582